test/file.jl

```diff
@@ -1141,7 +1141,7 @@ if !Sys.iswindows()
         fn = tempname()
         run(`mkfifo $fn`)
 
-        script = "x = open(\"$fn\", \"w\"); close(x)"
+        script = "x = open(\"$fn\", \"w\"); sleep(0.1); close(x); quit()"
```

Comment:
while we're changing this test, can you replace `\"$fn\"` with `$(repr(fn))` for me too? (that'll make it robust against other random meta-characters happening to to get inject into temp path)